### ISSUE TITLE
The "defense" dropdown should maybe be scrollable

### ISSUE THREAD

**Issue by Niko-Sn** [DecentralCardGame/frontend#313]
![image](https://github.com/user-attachments/assets/0e571428-3b04-401d-96bf-e69b7c52bab5)

Maybe other dropdowns have same issue?

**Comment by Niko-Sn** [comment]
(this is in card creator)

**Comment by Niko-Sn** [comment]
This is an issue on almost all dropdowns for me in card creator, where the list is big enough to where it exceeds the page height.
Please make this dropdown scrollable

**Comment by patrickwieth** [comment]
It was a conscious design choice to keep these dropdowns minimal. (and not introduce scrollbars inside)
I agree some solution for this must be found. @AnnyBuh

**Comment by Niko-Sn** [comment]
Okay, can we have a scrollbar in the meantime?

**Comment by patrickwieth** [comment]
Nope, I don't think it makes sense to invest time in something that is temporary, just looks nicer (functionality works) and actually you dont pick high numbers anyway here.
